x-pack/plugins/serverless_search/public/application/components/overview.tsx

```diff
@@ -37,6 +37,7 @@ import type {
 } from '@kbn/search-api-panels';
 import { useLocation } from 'react-router-dom';
 import { CloudDetailsPanel } from '@kbn/search-api-panels';
+import { apiService } from '@kbn/ingest-pipelines-plugin/public';
```

Comment:
see other comment, but we should create an api and [api hook](https://github.com/elastic/kibana/tree/main/x-pack/plugins/serverless_search/public/application/hooks/api) to get this data to follow the conventions in this plugin.

Reply:
Updated